core/crypto/sm3.c

```diff
@@ -50,6 +50,19 @@ void sm3_init(struct sm3_context *ctx)
 	ctx->state[7] = 0xB0FB0E4E;
 }
 
+#define SHL(x, n)	((x) << (n))
+static uint32_t rotl(uint32_t val, int shift)
```

Comment:
add an empty line above?
Or move the macro definition into the function body (only used there).

Reply:
I will add an empty line then since https://github.com/OP-TEE/optee_os/pull/6591#discussion_r1448575965